flake.nix

```diff
@@ -33,26 +61,9 @@
         };
 
         packages = rec {
-          xwayland-satellite = naersk'.buildPackage {
-            src = ./.;
-
-            nativeBuildInputs = with pkgs; [
-              rustPlatform.bindgenHook
-              rust-bin.stable.latest.default
-              pkg-config
-
-              xcb-util-cursor
-              xorg.libxcb
-
-              makeWrapper
-            ];
-            buildInputs = [ pkgs.xwayland ];
-
-            postInstall = ''
-              wrapProgram $out/bin/xwayland-satellite \
-                --prefix PATH : ${pkgs.xwayland}/bin
-            '';
-          };
+          xwayland-satellite-nosd = buildXwaylandSatellite { withSystemd = false; };
```

Comment:
Why is this a separate package output, rather than using `pkgs.callPackage` to make it overridable [like is done in nixpkgs](https://github.com/NixOS/nixpkgs/blob/d084336c705fb55a6c0daa247adb2d99d091810c/pkgs/by-name/xw/xwayland-satellite/package.nix#L11)? That way, `packages.<system>.xwayland-satellite-nosd` would instead be spelled `packages.<system>.xwayland-satellite.override { withSystemd = false; }`.

Reply:
> I think nixpkgs is downstream from us.

Good point, you got it. 

I've opened a PR (#65) addressing the mentioned changes, feel free to review it.